frontend/src/lib/components/ui/sparkline/sparkline.svelte

```diff
@@ -4,7 +4,6 @@
     ScaleTypes,
     type AreaChartOptions,
   } from '@carbon/charts-svelte';
-  import '@carbon/charts-svelte/styles.css';
```

Comment:
How come we removed them from here?

Reply:
It was included multiple places, now it is imported once